examples/run_fine_tuning.py

```diff
@@ -325,13 +334,34 @@ def export_model_preds(preds_file, predictions, validation_stream):
 
     print("Generated text: ", prediction_results)
 
+    if args.tgis:
+
+        # Saving model
+        model.save(args.output_dir)
+
+        # Load model in TGIS
+        # HACK: export args.output_dir as MODEL_NAME for TGIS
+        # container to pick up automatically
+        os.environ["MODEL_DIR"] = os.path.dirname(args.output_dir)
```

Comment:
I might be missing it, but what is `MODEL_DIR` used for? This isn't a TGIS env var, right?